backend/src/main/java/hu/flowacademy/vajdasagbrand/controller/UserController.java

```diff
@@ -29,13 +33,26 @@ public AccessTokenResponse login(@RequestBody LoginDto loginDto) {
         return keycloakClientService.login(loginDto.getUsername(),loginDto.getPassword());
     }
 
+    @RolesAllowed("SuperAdmin")
+    @DeleteMapping("/delete-user/{id}")
```

Comment:
Legyen egységes, használd a `users` itt is

Reply:
átírtam